### PR TITLE
Remove BreakingChangeAnalysis

### PR DESCRIPTION
**Pull request by pragnagopa**
Auto-upgrade completed. Removing breaking change analysis to avoid noise in the logs

### CODE REVIEW

**Comment by pragnagopa** [comment]
cc @FinVamp1

**Comment by fabiocav** [comment]
The dependency set for this needs to be updated.

Here are the failures:

Changed:
- protobuf-net.dll: 3.0.0.0/3.0.0.0 -> 2.3.3.0/2.3.3.0

Removed:
- DotNetTI.BreakingChangeAnalysis.dll: 1.1.0.0/1.1.0.0
- Marklio.Metadata.dll: 1.2.20.0/1.2.20.0
- protobuf-net.Core.dll: 3.0.0.0/3.0.0.0
- System.Private.ServiceModel.dll: 4.6.0.0/4.600.19.47001
- System.ServiceModel.dll: 4.0.0.0/4.600.19.47001
- System.ServiceModel.Primitives.dll: 4.6.0.0/4.600.19.47001


Aside from `Marklio.Metadata.dll` and  `DotNetTI.BreakingChangeAnalysis.dll`, we should keep the other dependencies as-is. 

We also need to update the runtime assemblies manifest

**Review by fabiocav — CHANGES_REQUESTED** [review]
The dependency set for this needs to be updated.

Here are the failures:

Changed:

protobuf-net.dll: 3.0.0.0/3.0.0.0 -> 2.3.3.0/2.3.3.0
Removed:

DotNetTI.BreakingChangeAnalysis.dll: 1.1.0.0/1.1.0.0
Marklio.Metadata.dll: 1.2.20.0/1.2.20.0
protobuf-net.Core.dll: 3.0.0.0/3.0.0.0
System.Private.ServiceModel.dll: 4.6.0.0/4.600.19.47001
System.ServiceModel.dll: 4.0.0.0/4.600.19.47001
System.ServiceModel.Primitives.dll: 4.6.0.0/4.600.19.47001
Aside from Marklio.Metadata.dll and DotNetTI.BreakingChangeAnalysis.dll, we should keep the other dependencies as-is.

We also need to update the runtime assemblies manifest

File: test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json [hunk 1]
```diff
@@ -423,21 +422,6 @@
           "lib/netstandard2.0/Azure.Storage.Common.dll": {}
         }
       },
-      "DotNetTI.BreakingChangeAnalysis/1.1.0-preview": {
-        "dependencies": {
-          "Marklio.Metadata": "1.2.20-beta",
```
Comment (pragnagopa) [inline test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json:428]: @brettsam  - please do look at the changes in this file. Had to update as the test caught the changes

Comment (fabiocav) [reply test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json:428]: Added comments about this above.

Comment (pragnagopa) [reply test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json:428]: Thanks! Just saw this. Addressing in the next iteration.

Comment (pragnagopa) [reply test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json:428]: Moving `protobuf-net` from `"3.0.0-alpha.122` to `3.0.0` .  Let me know if you have concerns with this

File: src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj [hunk 2]
```diff
@@ -73,6 +72,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging" Version="4.0.2" />
     <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.91-alpha" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.3" />
+    <PackageReference Include="protobuf-net" Version="3.0.0" />
```
Comment (brettsam) [inline src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj:75]: Did one of the breaking change dependencies bring this in?

Comment (pragnagopa) [reply src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj:75]: That is right! `DotNetTI.BreakingChangeAnalysis` had dependency on this package. Added it back to ensure dependencies do not change

**Review by brettsam — APPROVED** [review]

**Comment by pragnagopa** [comment]
@fabiocav  - fyi merging as the feedback is addressed.

**Pull request merged**